modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager, you can grant access to applications that want to consume your APIs by creating instance Contracts. +
+After you grant access to applications in Exchange, a contract is automatically created for the API instance. You can then manage these contracts from API Manager. +
```

Comment:
"You can then manage these contracts from API Manager" May be from your API in API Manager? Contracts "live" within APIs.

Reply:
Fixed! Thanks!